CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing to config-lint
+
+Help wanted! We'd love your contributions to config-lint. Please review the following guidelines before contributing. Also, feel free to propose changes to these guidelines by updating this file and submitting a pull request.
+
+- [I have a question](#questions)
+- [I found a bug](#bugs)
+- [I have a feature request](#features)
+- [I have a contribution to share](#process)
+
+## <a name="questions"></a> Have a Question?
+
+Please don't open a GitHub issue for questions about how to use config-lint, as the goal is to use issues for managing bugs and feature requests.
```

Comment:
I don't think we have a good solution for this, but maybe we should at least direct people to the contact-us page at stelligent.com.

Reply:
Oh yes I meant to put some kind of placeholder there. I will add that for now.